app/components/earthworks/code_snippet_link_component.rb

```diff
@@ -0,0 +1,9 @@
+module Earthworks
+  # Display other options section in sidebar
```

Comment:
This comment seems copied from that component, possibly could confuse

Reply:
Will change to reflect what the component does